src/pudl/transform/ferc1.py

```diff
@@ -355,6 +362,71 @@ def align_row_numbers_dbf(
     return df
 
 
+class SelectDbfRowsFromCategory(TransformParams):
```

Comment:
`SelectDbfRowsByCategory`? See note below.

Reply:
again, you can always communicate one name change request or misspelling. in our convention here, changing the function name implies changing all these other names.